src/common/valgrind/drd.h

```diff
@@ -355,7 +355,7 @@
  * be reported.
  */
 #define ANNOTATE_BENIGN_RACE(addr, descr) \
-   ANNOTATE_BENIGN_RACE_SIZED(addr, sizeof(*(addr)), descr)
```

Comment:
IMHO this is not necessary, please see: 87d8a6f13863332dd4d50abe920aed36d5898b48